### PR TITLE
Update homepage button placement

### PR DESCRIPTION
**Pull request by jbalboni**
## Description

This moves the three colored "buttons" below the hub list and updates their styling to flow better between our different breakpoints. Previously, wrapping didn't work all that well and they were stacked vertically earlier than necessary. I also updated some markup to be more semantic, i.e. making the first box a link and the other two boxes buttons.

## Testing done
Tested locally at lots of screen sizes. Checked in IE 11.

## Screenshots

localhost_3001_ 3

localhost_3001_ ipad 3

localhost_3001_ pixel 2 xl 3

## Acceptance criteria
- [x] Buttons look correct at each breakpoint

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/sass/brand-consolidation.scss [hunk 1]
```diff
@@ -34,6 +34,7 @@ $image-path: "~uswds/src/img";
 
 @import "~@department-of-veterans-affairs/formation/sass/shame";
 @import "merger";
+@import "modules/m-homepage";
```
Comment (jbalboni) [inline src/platform/site-wide/sass/brand-consolidation.scss]: Started trying to pull some styles out into another file. I have another homepage ticket where I'm going to finish moving things, I think.

**Review by amycesal — APPROVED** [review]
Looks good!

We are also talking about left aligning the text for these buttons, which would help them be more legible when they wrap to 3 lines.

**Comment by jbalboni** [comment]
I updated this to left align the butttons:

screen shot 2018-10-02 at 10 41 39 am

I'm going to leave the old screenshots, since it's a minor change.

**Review by ncksllvn — APPROVED** [review]
LGTM

**Pull request merged**